src/components/KebabDropdown/KebabDrowdown.tsx

```diff
@@ -10,50 +11,44 @@ interface MenuItem {
 }
 
 const WalletKebab = (props: { menuList: MenuItem[] }) => {
-  const [isKebabDropdownOpen, setIsKebabDropdownOpen] = useState<boolean>(false)
-  const [activeItemId, setActiveItemId] = useState<number>(0)
+  const [isKebabDropdownOpen, setIsKebabDropdownOpen] = useState<boolean>(true)
```

Comment:
setIsKebabDropdownOpen > setKebabDropdownOpen ? We talked about such naming yesterday.